public/content/about/index.md

```diff
@@ -95,9 +95,9 @@ This means the website needs to handle many different user journeys, from “a d
 
 ## Product roadmap {#roadmap}
 
-To make our work more accessible and to foster more community collaboration, the ethereum.org core team publishes an overview of our quarterly roadmap goals.
+To make our work more accessible and to foster more community collaboration, the ethereum.org core team publishes an overview of our cyclical roadmap goals.
```

Comment:
```suggestion
To make our work more accessible and to foster more community collaboration, the ethereum.org core team publishes an overview of our [shape up cycle](https://www.productplan.com/glossary/shape-up-method/) roadmap goals.
```

Just making this a big more verbose so people can get some more information. But this looks good.